shopinvader_search_engine/component/shopinvader_variant_export_mapper.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# © 2018 Akretion (http://www.akretion.com)
+# Copyright 2018 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo.addons.component.core import Component
+
+
+class ShopinvaderVariantExportMapper(Component):
+    _name = 'shopinvader.variant.json.export.mapper'
+    _inherit = 'json.export.mapper'
+    _apply_on = ['shopinvader.variant']
+    _usage = 'se.export.mapper'
+
+    def _apply(self, map_record, options=None):
+        values = super(ShopinvaderVariantExportMapper, self)._apply(
+            map_record=map_record, options=options)
+        values.update(self._add_seo_title(map_record.source))
+        return values
+
+    def _add_seo_title(self, record):
```

Comment:
As discuss in the code sprint, it will be better to have two field
- manual_seo_title
- seo_title ( a computed field that use a defualt value or a manual version)
Indeed with the current solution if we remove the "seo_title" in the exporter or if we modify the exporter we will have a weird behavoiour because the use think he can change it but in reality it do not change anything as the mapper overright it